src/main/kotlin/com/sovdee/skriptbounds/bukkit/BukkitConversion.kt

```diff
@@ -13,7 +13,8 @@ fun toPosition(any: Any?) : Vector3d? {
 }
 
 fun toPosition(vector: Vector?) : Vector3d? {
-    return vector?.toVector3d()
+    val v = vector?.toVector3d()
+    return if (v?.isFinite == true) v else null
 }
```

Comment:
This could be a receiver, though it's unfortunate that there's already a toVector3d() on that vector.
```kt
fun Vector?.toPosition() {
    val v = vector?.toVector3d()
    return if (v?.isFinite == true) v else null
}
```
To clarify, this can still be called on nullables